deploy/testenv-deploy.js

```diff
@@ -0,0 +1,23 @@
+const env = require('./src/loadEnv')
+
+const deployToken = require('./src/utils/deployERC20Token')
+
+const mode = process.argv[2]
+
+async function main() {
+  //If other components of testing env needs to be initialized
```

Comment:
//If other components of the test env need .....

Reply:
I have removed this code comment since it is outdated.